code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -0,0 +1,239 @@
+
+#define BEEBOX_MAX_FRAMES				3		//Max frames per box
+#define BEES_RATIO						0.5 	//Multiplied by the max number of honeycombs to find the max number of bees
+#define BEE_PROB_NEW_BEE				20		//The chance for spare bee_resources to be turned into new bees
+#define BEE_RESOURCE_HONEYCOMB_COST		100		//The amount of bee_resources for a new honeycomb to be produced, percentage cost 1-100
+#define BEE_RESOURCE_NEW_BEE_COST		50		//The amount of bee_resources for a new bee to be produced, percentage cost 1-100
+
+
+
+/mob/proc/bee_friendly()
+	return 0
+
+
+/mob/living/simple_animal/hostile/poison/bees/bee_friendly()
+	return 1
+
+
+/mob/living/carbon/human/bee_friendly()
+	if(dna && dna.species && dna.species.id == "pod") //bees pollinate plants, duh.
+		return 1
```

Comment:
The way I was saying it could be done was to go to /datum/species/pod and add this line
`ignored_by = list(/mob/living/simple_animal/hostile/poison/bees)`
This works too, but adds a spare dna check. Either way works I suppose